go/skaffold.yaml

```diff
@@ -94,6 +94,14 @@ profiles:
           docker:
             buildArgs:
               GOVERSION: 1.19
+      - op: add
+        path: /build/artifacts/-
+        value:
+          image: go120app
+          context: test/goapp
+          docker:
+            buildArgs:
+              GOVERSION: '1.20'
```

Comment:
The quotes are necessary. Without them the trailing zero is lost and the buildarg becomes `1.2`.